lib/monitoring/aws-apigateway/ApiGatewayMonitoring.ts

```diff
@@ -115,48 +142,69 @@ export class ApiGatewayMonitoring extends Monitoring {
       scope.createMetricFactory(),
       props
     );
+
     this.tpsMetric = metricFactory.metricTps();
-    this.p50LatencyMetric = metricFactory.metricLatencyP50InMillis();
-    this.p90LatencyMetric = metricFactory.metricLatencyP90InMillis();
-    this.p99LatencyMetric = metricFactory.metricLatencyP99InMillis();
+
+    this.latencyMetrics = {};
+    this.latencyTypesToRender = [];
+    if (props.latencyTypesToRender) {
+      // try adding latency types specified by user first
+      props.latencyTypesToRender.forEach((type) =>
+        this.latencyTypesToRender.push(type)
+      );
+    } else {
+      // fallback to default latency types if user does not specify any
+      DefaultLatencyTypesToRender.forEach((type) =>
+        this.latencyTypesToRender.push(type)
+      );
+    }
```

Comment:
How about we simply do this?

```suggestion
    this.latencyTypesToRender = props.latencyTypesToRender ?? DefaultLatencyTypesToRender;
```

Reply:
Could potentially just do an array spread.